Mejoras tasas_vitales()

dianapg8 opened the pull request:
Agrego opción para acceder a datos unicamente estatales sin la cve_municipio
No se como descartar los demas commits oops

dianapg8 committed:
Update Inegi.jl

dianapg8 committed:
Update index.md

dianapg8 committed:
Mejoras tasas_vitales()

Agrego opción para cosultar unicamente datos por entidad

dianapg8 committed:
Merge pull request #3 from mucinoab/master

Actualización de documentación

mucinoab commented:
Para no traer otros commits, usas otra rama.

mucinoab reviewed (changes requested):
Falta dejar claro este nuevo comportamiento en la documentación.

dianapg8 committed:
Actualizo Documentación tasas_vitales()

Agrego descripción que indica la falta del parametro cve_municipio para acceder a datos estatales

dianapg8 commented:
Listo

> Falta dejar claro este nuevo comportamiento en la documentación.

Listo

mucinoab commented:
> Listo
> 
> > Falta dejar claro este nuevo comportamiento en la documentación.
> 
> Listo

Listo
